lsstswBuild.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
Can you add some commentary to the top of this file explaining your reasoning for flip-flopping the `e` flag continually through the code. It's asking for trouble in the future when someone edits this code to add an extra call somewhere.